lib/SPIRVProducerPass.cpp

```diff
@@ -440,7 +442,6 @@ struct SPIRVProducerPassImpl {
   SPIRVID getSPIRVType(Type *Ty);
   SPIRVID getSPIRVConstant(Constant *Cst);
   SPIRVID getSPIRVInt32Constant(uint32_t CstVal);
-  SPIRVID getSPIRVInt64Constant(uint64_t CstVal);
```

Comment:
Why do you want to remove this function?

Reply:
I thought I added it in myself for some reason